projects/incubator/README.md

```diff
@@ -0,0 +1,9 @@
+# Spartacus Incubator
+
+Spartacus Incubator is a package that include features still in development. The features included in the library can be used but are likelly to evolve in future releases and might even get refactored over time. The incubator package might contain core and storefront related code.
+
+Once a feature from the incubator library is ready for final release, it will get moved to the correct other Spartacus library.
+
+Each version of the incubator library has hard dependendies on an exact version of the other Spartacus libraries.
```

Comment:
Regarding versioning of the incubator package, we like the version to reflect the Spartacus release number, similar to how the CLI is doing this. 
For example, if we have spartacus release `1.4.0`, the incubator release would become `0.140.0`. Similarly, if we have a `1.5.0-next.0`, it would become `0.150.0-next.0`. 

The idea here is to:
- never release a major version 1. 
- always make a logical connection with the Spartacus release
- allow for patches on the incubator release (i.e. `0.140.1`)

It would be great to automate this in the release process. Not sure if you'd feel comfortable of doing so, or we if we do this in a separate ticket.

Reply:
Either way, I think this concept should become part of the readme.

Reply:
@hackergil @marlass appreciate your input here.

Reply:
It can be arranged to make the release process and version automated. Can you have an incubator patch release not tied to a patch release from other libs (i.e `0.140.1` with other libs at `1.4.0`)? If this is the case that means for Spartacus libs at `1.4.1` the incubator would be `0.141`?

Reply:
`0.141.0` is indeed tight to a `1.4.1` release of Spartacus, however in case a patch is required on the incubator, it would result in `0.141.1`.

Reply:
I don't think we need to automate it. It can be done, but I would for now keep it manual. Totally agree about documenting it not only in README, but also on our release wiki page.

Reply:
Will update the wiki page.